scripts/generate_image.ps1

```diff
@@ -0,0 +1,175 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# This script is used to generate windows docker images used by agent
+# integration tests and functions tests and upload images to ECR.
+
+Param(
+    [string]$GIT_REPO,
+    [string]$COMMIT_SHA,
+    [string]$S_REPOSITORY,
+    [string]$R_REPOSITORY,
+    [string]$S_REGION,
+    [string]$R_REGION
+)
+
+$ErrorActionPreference = 'Continue'
+
+# Constants
+$GitVersion="2.10.1"
```

Comment:
Since the gitversion and go version are static here, is there any reason we hit github every time rather than pulling things down once for our entire test run?